test/e2e_node/image_pull_test.go

```diff
@@ -197,7 +186,13 @@ var _ = SIGDescribe("Pull Image", feature.CriProxy, framework.WithSerial(), func
 
 			var pods []*v1.Pod
 			for _, testpod := range testpods {
-				pods = append(pods, e2epod.NewPodClient(f).Create(ctx, testpod))
+				pod := e2epod.NewPodClient(f).Create(ctx, testpod)
+				ginkgo.DeferCleanup(func() {
+					if pod != nil {
```

Comment:
See above.

Reply:
Thanks for explaining, done using the shorter version.